standalone/Microsoft.Private.Intellisense/xmldocs/netcoreapp/1033/System.Runtime.xml

```diff
@@ -20091,11 +20091,8 @@ The <see cref="P:System.Nullable`1.HasValue" /> property for the current <see cr
       <returns>The hash code.</returns>
     </member>
     <member name="M:System.Range.GetOffsetAndLength(System.Int32)">
-      <summary>Returns a <see cref="T:System.Range.OffsetAndLength" /> instance with the starting offset and length of a range.</summary>
-      <param name="length">A positive integer that represents the length of the collection that the range will be used with.</param>
-      <returns>The starting offset and length of the range.</returns>
-      <exception cref="T:System.ArgumentOutOfRangeException">
-        <paramref name="length" /> is outside the bounds of the current range.</exception>
+      <summary>Calculates the start offset and length of the range object using a collection length.</summary>
+      <param name="length">The length of the collection that the range will be used with. <paramref name="length" /> has to be a positive value.</param>
```

Comment:
```suggestion
      <param name="length">The length of the collection that the range will be used with. <paramref name="length" /> has to be a positive value.</param>
      <returns>The starting offset and length of the range.</returns>
      <exception cref="T:System.ArgumentOutOfRangeException">
        <paramref name="length" /> is outside the bounds of the current range.</exception>
```

Reply:
i'm updating this manually from my machine (the file is huge, github doesn't let me commit the suggestion).